assets/app/styles/_log.less

```diff
@@ -100,14 +90,16 @@
   }
 }
 .log-line-number {
-  .flex(@columns: 0 0 60px); // set width of number column
+  .unselectable();
+  border-right: 1px lighten(@log-bg-color, 10%) solid;
+  min-width: 60px;
```

Comment:
Need that much for 5 digits. I set that as a min to avoid the log content jumping to the right when we reach 100 or 1000 lines, but maybe it's not such a problem.